server/db/models/User.js

```diff
@@ -0,0 +1,25 @@
+const connection = require('../connection')
+const { Sequelize } = connection
+const { UUID, UUIDV4, STRING, INTEGER } = Sequelize
+
+module.exports = connection.define('user', {
+    id: {
+        type: UUID,
+        primaryKey: true,
+        defaultValue: UUIDV4
+    },
+    googleId: {
```

Comment:
what is the google id like? sorry, i wanted to run the app locally but i can't because i didn't have the id/secret...